recipes/pyfesom2/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "pyfesom2" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/FESOM/pyfesom2/archive/{{ version }}.tar.gz
+  sha256: 90363de4f6b78f9eec72ac1f030ae2561062cd9c05a7ce6736f886fd1cf6c347
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Do we need to skip building on windows? The README makes it look like windows is supported. If we drop this line, I think this might be able to use `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Will be hard to support since I don't have experience on Windows, but why not try :)